src/giskardpy/tree_manager.py

```diff
@@ -0,0 +1,140 @@
+from giskardpy.plugin import PluginBehavior
+from giskardpy import logging
+from sortedcontainers import SortedList
+
+class TreeManager():
```

Comment:
unless there is a specific reason for not doing, please inherit from `object` -> `class TreeManager(object):`